runtime/compiler/runtime/IProfiler.hpp

```diff
@@ -565,7 +565,9 @@ class TR_IProfiler : public TR_ExternalProfiler
    void setupEntriesInHashTable(TR_IProfiler *ip);
 
    TR_IPMethodHashTableEntry *findOrCreateMethodEntry(J9Method *, J9Method *, bool addIt, uint32_t pcIndex =  ~0);
-   uint32_t releaseAllEntries();
+   // Returns the number of entries released, and also adds the number of
```

Comment:
Suggestion: "// Returns the number of entries released, and also stores the number of"

Reply:
Fixed